webui/react/src/pages/ExperimentList.tsx

```diff
@@ -79,6 +132,69 @@ const ExperimentList: React.FC = () => {
     handleFilterChange({ ...filters, username });
   }, [ filters, handleFilterChange ]);
 
+  const getBatchActions = useCallback((action: Action): Promise<void[] | Command> => {
+    if (action === Action.OpenTensorBoard) {
+      return launchTensorboard({
+        ids: selectedExperiments.map(experiment => experiment.id),
+        type: TBSourceType.Experiment,
+      });
+    }
+    return Promise.all(selectedExperiments
+      .map(experiment => {
+        if (action === Action.Activate) {
+          return setExperimentState({ experimentId: experiment.id, state: RunState.Active });
+        } else if (action === Action.Archive) {
+          return archiveExperiment(experiment.id, true);
+        } else if (action === Action.Cancel) {
+          return setExperimentState({ experimentId: experiment.id, state: RunState.Canceled });
+        } else if (action === Action.Kill) {
+          return killExperiment({ experimentId: experiment.id });
+        } else if (action === Action.Pause) {
+          return setExperimentState({ experimentId: experiment.id, state: RunState.Paused });
+        }
+        return Promise.resolve();
+      }));
+  }, [ selectedExperiments ]);
+
+  const handleBatchAction = useCallback(async (action: Action) => {
+    try {
+      const result = await getBatchActions(action);
```

Comment:
blocker: once this resolves can we trigger the experiments rest context to fetch the latest state to get the updates a bit quicker. 
similar to this https://github.com/determined-ai/determined/pull/858/files?file-filters%5B%5D=.tsx#diff-afa4dcb145f465ed696ff6ee10e2781bR66

or another way to deliver the updated states